llvm/tools/llvm-gsymutil/llvm-gsymutil.cpp

```diff
@@ -96,6 +97,7 @@ static bool Quiet;
 static std::vector<uint64_t> LookupAddresses;
 static bool LookupAddressesFromStdin;
 static bool StoreMergedFunctionInfo = false;
+static std::vector<std::string> CallSiteYamlPaths;
```

Comment:
Now you use the last arg only.
```suggestion
static std::string CallSiteYamlPath;
```